### DISCUSSION TITLE
Archive user

### DISCUSSION THREAD

**Discussion by marshmallowlike** [kimai/kimai#3191] [Questions & Answers] [answered]
Hi,
is it possible to archive users anonymously? 

I still need the hours, but I am not allowed to save the name or email address of the person after the end of employment.
I thought about renaming them e.g. "Dummy Assistant", "Dummy Manager" etc, but the mail address would also have to be changed. I can't use the general admin mail address because a mail address can only be used once.

Thanks in advance for any advice.

**Comment by kevinpapst** [comment, marked as answer]
I don't exactly understand your issue here. The email address does not have to be existing, so you can type anything in there. 
There is even an official domain that was made for such test cases: 
`dummy1@example.com`, `dummy2@example.com`, `dummy3@example.com`

Would that be sufficient for your issue?

**Reply by marshmallowlike** [reply]
Thank you (also for the very quick answer)! I didn't think it was that simple. I had assumed that I had to use a "real" email address. I guess I just didn't realise the obvious.

**Reply by kevinpapst** [reply]
Happens to all of us 😁 glad it works for you
